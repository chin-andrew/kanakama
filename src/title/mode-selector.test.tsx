import { shallow } from 'enzyme';
import modeSelector from './mode-selector';

describe('mode selector tests', () => {
  const renderModeSelector = () => {
    return shallow(modeSelector(() => {}));
  }

  it('renders buttons for selecting a practice mode', () => {
    const wrapper = renderModeSelector();
    expect(wrapper.find('button')).toHaveLength(3)
  }) 
})