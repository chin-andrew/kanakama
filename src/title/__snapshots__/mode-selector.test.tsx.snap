// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mode selector tests renders buttons for selecting a practice mode 1`] = `
<div
  className="sc-bwzfXH dVfpzR"
>
  <span
    className="sc-htpNat joCuXP"
  >
    What would you like to practice?
  </span>
  <button
    className="sc-bdVaJa eYKQVy"
    onClick={[Function]}
  >
    Hiragana
  </button>
  <button
    className="sc-bdVaJa eYKQVy"
    onClick={[Function]}
  >
    Katakana
  </button>
  <button
    className="sc-bdVaJa eYKQVy"
    onClick={[Function]}
  >
    Hiragana and Katakana
  </button>
</div>
`;
